SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -426,22 +428,19 @@ There are five right-handed Cartesian coordinate systems (<<coordinateSystemList
 |===
 |Coordinate system name|Origin|_x_-axis|_y_-axis|_z_-axis
 
-|Fixed coordinate system|Fixed somewhere on the nominal sea surface|Pointing North|Pointing East|Pointing down with gravity
 |Surface coordinate system|Platform origin|Forward along horizontal projection of platform main axis|Pointing horizontally to starboard orthogonal to horizontal projection of platform main axis|Pointing down with gravity
```

Comment:
|Surface coordinate system|Platform origin|Pointing North|Pointing East|Pointing down with gravity
comment: the transformation from SCS to PCS are straigthforward using the 3 rotation angles given by the attitude sensor, the transformation from Geographical to SCS is straigthfroward using two rotations with latitude and longitude

Reply:
Agreed. There is a detail about the angles that I will add soon (the angles from attitude sensors are not necessarily the angles that are meant by pitch and roll - the intrinsic/extrinsic thing).